### DISCUSSION TITLE
Bots fail to navigate back into the submarine from outside

### DISCUSSION THREAD

**Discussion by NovaStar987** [FakeFishGames/Barotrauma#14566] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Bot-controlled humans are notr able to navigate back into the submarine from outside. (singleplayer)

### Reproduction steps

Have a bot outside the submarine (preferably in a cave from a mining mission), then switch to another human in the sub.

In previous version, if you switch to another human, that bot would automatically navigate back into the submarine. However, now it just stays in place, even with the "Return to Submarine" command active.

EDIT: It turns out that this is actually due to the fact that I was using a custom submarine, where a custom (no buttons) hatch leads to the outside. Is there any way to fix this thing?

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.5.9.2 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by SomeRandomNoobKekeke** [comment]
I tried in quickstart and bots always return to the sub as they should

Devs didn't change anything that could cause this in 1.5.9.2 https://github.com/FakeFishGames/Barotrauma/commit/e8920829c91179241be06948ca11540b3c7a7a66

**Comment by NovaStar987** [comment]
It turns out that this is actually due to the fact that I was using a custom submarine, where a custom (no buttons) hatch leads to the outside. Is there any way to fix this thing?

**Comment by NovaStar987** [comment]
Resolution Found:
Custom Hatch needs to be linked (spacebar) to airlock and access buttons. 
Devs please close this ticket
